src/Core/Auth/Models/Data/IGrant.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Bit.Core.Auth.Models.Data;
+
+public interface IGrant
```

Comment:
ℹ️  Our SQL storage of grants now has an integer ID as a primary key -- may want to represent this here.

Reply:
I left that on the Grant entity use for SQL since it didn't seem necessary on other implementations